Update readme to add in context for fork (#1)
alphagov/accessible-autocomplete-multiselect: README.md

[![Sauce Labs Build Status](https://saucelabs.com/browser-matrix/tvararu-alphagov.svg)](https://saucelabs.com/u/tvararu-alphagov)

> This is a fork of the [accessible-autocomplete](https://github.com/alphagov/accessible-autocomplete) module built and maintained by the GOVUK Design System team.
> 
> This fork adds additional functionality such as support for [select multiple](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attr-multiple).

Reviewer: I think the _"built and maintained by the GOVUK Design System team"_ wording might be a bit confusing. I initially understood it to mean that _this forked version_ is maintained by that team.

I think something like this could be more succinct:

```suggestion
This is a fork of the [accessible-autocomplete](https://github.com/alphagov/accessible-autocomplete) component. It adds additional functionality such as support for [select multiple](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#attr-multiple).
```

What do you think?

Author: Sounds good
